galasa-eclipse-parent/pom.xml

```diff
@@ -78,6 +78,12 @@
 				<artifactId>dev.galasa.framework</artifactId>
 				<version>0.15.0-SNAPSHOT</version>
 			</dependency>
+			<dependency>

```

Comment:
I would suggest adding this to the actual project not parent.    Have you researched what level will be included in Eclipse Oxygen?